src/globals.ts

```diff
@@ -43,6 +43,35 @@ export const options: GlobalOptions = {
     timestamp: 0,
 };
 
+export const eventEmitter = new EventEmitter();
+export const minimapEvents = (() => {
+    let intervalId: NodeJS.Timer;
+    return {
+        startInterval: () => {
+            const interval = options.samplingTime * 1_000;
```

Comment:
Can we have a maximum time? So that it at least updates every 3 seconds or something?